### DISCUSSION TITLE
Supported File Formats

### DISCUSSION THREAD

**Discussion by keithpij** [Filimoa/open-parse#14] [Q&A] [answered]
What file formats are supported? It looks like PDFs are supported, but are other popular formats, such as .doc, .docx, .xls, .xlsx, etc., also supported?

**Comment by Filimoa** [comment, marked as answer]
Currently just pdf - we may extend this to .doc + .docx + .ptt + .pptx + .md soon. I'm curious what's your use cases for excel files? I honestly don't think you'd want to use a layout parser on them - it's inherently lossy.

**Reply by keithpij** [reply]
I do not have a specific use case. I am doing research on libraries that can be used within the RAG pipeline and came across your project.
